lessons/02-javascript/INCLASS.md

```diff
@@ -0,0 +1,34 @@
+# Lesson 2 In-Class Exercises
+(As before, we’re not expecting you to finish all of these activities! Pick the ones which are interesting and challenging to you, and feel free to work with the people around you.)
+
+## JavaScript Concepts:
+- Break into groups of ~4 members. Each member is in charge of finding out about a specific javascript concept and writing a small jsfiddle that demonstrates it
+  - Comeback together and give quick whiteboard talks about your concepts
+  - Concepts:
+    - First-Class and Higher Order Functions
+    - Prototypes/Inheritance
+	- Bind, Apply, Call
+    - Closures
+
+## Event Listeners:
+- Much of Web development is learning to react to event, as first foray into this write a small codepen [like this](http://codepen.io/jagreene/pen/pjMRvb) that creates a simple html element and responds to an event of your choice.
+
+## Functional Programming:
+- Part of the power of JavaScript is that is supports imperative (like C), object oriented (like C++ or Java), and functional (like Fortran or Lisp) programming paradigms. Its likely you've run into the first two through sofdes, so if you are new to functional programming now is a great time to get an introduction
```

Comment:
- IT supports
- SoftDes (or "previous software classes")
- , BUT if you are new... introduction.